### DISCUSSION TITLE
Add Search bar in my Backpack

### DISCUSSION THREAD

**Discussion by Alex-H-on-GitHub** [ScratchAddons/ScratchAddons#7675] [Ideas]
I have a really BIG backpack, and I'm talking about **hundreds** of sprites. Now I don't want to waste my time scrolling down and finding sprites. So I thought of an idea to add a search bar to your back pack, if it's capable, and it works similar to the "Search bar on My Stuff" addon, but it's in your backpack.

This here is my concept of what it would look like. And before you ask, **those are question boxes made by me**, like I'm **not** having a serious bug that corrupts my backpack or something.

<img width="1200" alt="Screenshot 2024-07-23 at 10 20 55 PM" src="https://github.com/user-attachments/assets/823aa53b-83a7-4185-841f-27911793a70d">
(I made this with concept with Scratch/TurboWarp)

Oh yeah and this is my first discussion on ScratchAddons.

**Comment by WorldLanguages** [comment]
Congrats on your first discussion

This is being tracked in the following issue: #7435
